android/ReactAndroid/build.gradle

```diff
@@ -345,6 +345,8 @@ def packageReactNdkLibsForBuck = tasks.register("packageReactNdkLibsForBuck", Co
 android {
     compileSdkVersion 28
 
+    resourcePrefix 'reactandroid_'
```

Comment:
do you know if there's a way to lint the style/theme names, which seem to be in CamelCase, as well? or do those just need manual checks?

Reply:
Actually, Android Studio is pretty smart about this and doesn't consider a violation names like:
- `ReactAndroidRedboxButton` (even though camelcase of `reactandroid_` would be `Reactandroid`!)
- `reactandroid_catalyst_debug_error`